### PR TITLE
Added: Toast Component [P1]

### PR DESCRIPTION
**Pull request by samflab**
please review this component. thank you

### CODE REVIEW

**Comment by narharikale** [comment]
seems good to me 👍

**Review by NtshVrm — APPROVED** [review]
Looks good👍

File: toast/toast.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const baselineBtn = document.querySelector(".baseline-btn");
+const baselineToast = document.querySelector(".baseline-toast");
+const toastBtn = document.querySelector(".toast-btn");
+
+const hideToast = () => (baselineToast.style.visibility = "hidden");
+const showToast = () => {
+  baselineToast.style.visibility = "visible";
+  setTimeout(hideToast, 5000);
```
Comment (shraddha15-creator) [inline toast/toast.js]: I feel 5 seconds are too much. You can make it 3 seconds.

**Review by shraddha15-creator — CHANGES_REQUESTED** [review]

File: toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  font-size: 1.1rem;
+  margin-right: 5px;
+  color: var(--font);
+}
+
+.toast-btn {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  font-size: medium;
```
Comment (rutvikpumak) [inline toast/toast.css]: give exact font-size

File: toast/toast.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
+@import url("../styles.css");
+
+.baseline-btn {
+  background-color: var(--primary-dark);
+  border: none;
+  border-radius: 5px;
+  color: var(--font);
+  cursor: pointer;
+  font-size: 1.1rem;
+  font-weight: 600;
```
Comment (rutvikpumak) [inline toast/toast.css]: make a local variable for font-weight in the main CSS file

File: toast/toast.js (hunk 1 above)
Comment (samflab) [reply toast/toast.js]: okay noted !

**Review by shraddha15-creator — APPROVED** [review]

File: toast/toast.css (hunk 2 above)
Comment (samflab) [reply toast/toast.css]: done

File: toast/toast.css (hunk 3 above)
Comment (samflab) [reply toast/toast.css]: done

**Pull request merged**